setup.cfg

```diff
@@ -30,6 +30,7 @@ install_requires =
     pymongo
     motor
     requests
+    httpx
```

Comment:
My CI was failing because they couldn't find httpx - see the first commit on this PR. Didn't know how else to solve it... maybe I could add it to requirements.txt?

Reply:
Hmm, reason I asked was because httpx and requests do basically the same thing, so it seems a shame to depend on both. I might have a play with it and see why that is.

Reply:
At the very least though I'd recommend adding to dev requirements rather than main requirements if it's only needed for CI

Reply:
OK, sure. I'll add that in

Reply:
But yes I'm also confused, as it's a relatively recent problem - never had the CI fail for that reason before :thinking:

Reply:
Resolving as I've moved httpx to be a dev dependency. Will make a PR in future to update to latest version of skeleton which I think will make my dependencies cleaner.